src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -3124,7 +3145,21 @@ public enum CondemnedReasonGroup
         Too_Small_For_BGC = 12,
         Ephemeral_Before_BGC = 13,
         Internal_Tuning = 14,
-        Max = 15,
+        Almost_max_alloc = 15,
```

Comment:
nit - could we please following the style here for cases like the existing code?